### ISSUE TITLE
Lower shield resource pack

### ISSUE THREAD

**Issue by RedCarlos26** [alt03b1/BedrockVanillaTweaks#6]
Hello, the guy that does the lower shield resource pack did not update his pack to the newer versions (1.16.1), so it makes that we see the pigling as a zombie pig man. Will it have an update soon?

**Comment by lithium-lx** [comment]
I will message him and update this issue when it's fixed!

**Comment by RedCarlos26** [comment]
Thanks! Looking for it!

**Comment by RedCarlos26** [comment]
Any news?

**Comment by lithium-lx** [comment]
Sady I have not recieved anything from the texture designer. Since I don't know how that works. I can't give you any futher support.
Sorry

**Comment by KaanGaming** [comment]
I see the "zombie pigman" you mentioned as zombified piglin.

![](https://i.ibb.co/cr2FdWn/Screenshot-20200819-215956.jpg)

I am using the Lower Shield resource pack as you used, and there is no zombie pigmen texture as you mentioned.
Have you ever considered updating Minecraft? If not, what version are you using?

---

#### Here is additional information (or screenshots) if you care.
###### This is contents of Lower Shield pack. First one is the textures, the second one is the models.
###### (Ignore the scribble)
![](https://i.ibb.co/3TbbC8D/scree1.png)
![](https://i.ibb.co/b10YSXC/scree2.png)

**Comment by RedCarlos26** [comment]
No, I didn't had it on. I never even installed it. The guy said that he needed to update all he's packs to fix the issue.

**Comment by KaanGaming** [comment]
I have no leads. This seems like a dead end, but it cannot be one.

Here I published the zip file of the .mcpack, it was simple to do because it was one.
I unarchived it with 7-zip and archived it back to .zip.
[amybluesky addon_lower shield.zip](https://github.com/alt03b1/BedrockVanillaTweaks/files/5102152/amybluesky.addon_lower.shield.zip)

Feel free to explore the pack.
The key points are definitely `manifest.json`, `textures` folder and `models` folder. I mean, there are no any other folders other than that.

---

For those who are novice at this, manifest.json contains 5 or 6 key points in the file.
- description, name, version (metadata)
- uuid (unique pack id)
- min_engine_version (seems like a minimum version requirement)
- type (type of the pack)

Rest are self explanatory (by "rest", that means the folders, files, and other stuff)
